R/carehomes.R

```diff
@@ -1414,17 +1691,3 @@ carehomes_data <- function(data, start_date, dt) {
   data <- sircovid_data(data, start_date, dt, expected)
   carehomes_particle_filter_data(data)
 }
-
-
-scale_severity <- function(severity, strain_rel_severity,
-                           which = c("p_G_D", "p_H_D", "p_W_D", "p_ICU_D")) {
-  severity[which] <- lapply(severity[which], function(x) {
-    x <- matrix(x, nrow = length(x), ncol = length(strain_rel_severity))
-    prob <- matrix(strain_rel_severity, nrow = nrow(x),
-                  ncol = length(strain_rel_severity), byrow = TRUE)
-    x <- x * prob
-    x[x > 1] <- 1
```

Comment:
I can see that we don't need a separate function for these now as it's been generalised but do we have a check like this somewhere else to make sure no probs are above 1?